db/migrate/20170405065305_remove_image_password_audit.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveImagePasswordAudit < ActiveRecord::Migration
+  def up
+    Audited::Audit.where(:auditable_type => 'Image').find_in_batches do |batch|
+      batch.each do |audit|
+        audit.audited_changes.delete("password")
+        if audit.changed?
```

Comment:
it won't work reliably since it will also detect audits where other attributes contain value password but it limits some records, I'll update it